openraft/src/raft.rs

```diff
@@ -399,6 +407,131 @@ impl<C: RaftTypeConfig, N: RaftNetworkFactory<C>, S: RaftStorage<C>> Raft<C, N,
         Ok(res)
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
+    pub async fn remove_members(
```

Comment:
The body of `change_membership()`, `remove_members()` and `add_members()` are almost the same.
The common part can be extracted into another method to reduce duplicated codes.